src/commands/set/secret-account.ts

```diff
@@ -0,0 +1,59 @@
+import { Input, Select } from 'cliffy/prompt/mod.ts';
+import { SupportedProviders } from '../../@providers/index.ts';
+import { BaseCommand, CommandHelper, GlobalOptions } from '../base-command.ts';
+
+type SetSecretAccountOptions = {
+  provider?: string;
+  namespace?: string;
+  creds?: string[];
+} & GlobalOptions;
+
+const SetSecretAccountCommand = BaseCommand()
+  .description('Configure where to store the configuration settings for arcctl')
+  .option('--provider <provider:string>', 'Which provider type to use')
+  .option('--namespace <namespace:string>', 'The namespace to use for the prvoider')
+  .option('--creds <creds:string>', 'A key value pair of credentials to use for the provider', { collect: true })
+  .action(set_secret_account);
+
+async function set_secret_account(options: SetSecretAccountOptions) {
+  const command_helper = new CommandHelper(options);
+
+  const providerName = options.provider ||
+    (await Select.prompt({
+      message: 'What provider will this account connect to?',
+      options: Object.keys(SupportedProviders),
+    }));
+
+  const providerType = providerName as keyof typeof SupportedProviders;
+
+  const providedCredentials: Record<string, string> = {};
+  for (const cred of options.creds || []) {
+    const [key, value] = cred.split('=');
+    providedCredentials[key] = value;
+  }
+
+  const credentials = await command_helper.promptForCredentials(providerType, providedCredentials);
+
+  const namespace = options.namespace || await Input.prompt({
+    message: 'What namespace should this account use?',
+  });
+
+  const account = new SupportedProviders[providerType](
+    'secret',
+    credentials as any,
+    command_helper.providerStore,
+    {},
+  );
+  const validCredentials = await account.testCredentials();
+  if (!validCredentials) {
+    throw new Error('Invalid credentials');
+  }
+
+  await command_helper.secretStore.save({
+    provider: providerType,
+    credentials,
+    namespace,
+  });
```

Comment:
Why is this a "store" anymore? Isn't this more like CLI configuration and it will always be stored on the filesystem of the CLI?

Reply:
We do not really have a config file for the application, though we should probably change that.

Reply:
I agree. It would definitely be helpful to have.